src/sentry/models/organizationmemberreplayaccess.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_model, sane_repr
+
+
+@region_silo_model
+class OrganizationMemberReplayAccess(Model):
+    """
+    Tracks which organization members have permission to access replay data.
+
+    When no records exist for an organization, all members have access (default).
+    When records exist, only members with a record can access replays.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    organization = FlexibleForeignKey("sentry.Organization", related_name="replay_access_set")
+    organizationmember = FlexibleForeignKey(
+        "sentry.OrganizationMember",
+        on_delete=models.CASCADE,
+        related_name="replay_access",
+    )
```

Comment:
Do we need both of these? We can join to `organizationmember` to get the organization